capstone-frontend/src/styles/SearchUserModal.css

```diff
@@ -0,0 +1,7 @@
+.margin-auto {
+  margin: auto
+}
+
+.email-text {
+  font-size: 0.9rem !important;
```

Comment:
Try using an id selector here so you can get rid of `!important`

Reply:
Done. I forgot that the id selector can override classes.